OptionSelector/DetailsTableViewController.m

```diff
@@ -0,0 +1,166 @@
+//
+//  DetailsTableViewController.m
+//  OptionSelector
+//
+//  Created by Diana Elezaj on 8/13/15.
+//  Copyright (c) 2015 Diana Elezaj. All rights reserved.
+//
+
+#import "DetailsTableViewController.h"
+#import "AllCategoriesTableViewController.h"
+#import "CQCategory.h"
+#import <AVFoundation/AVFoundation.h> // this allows us to include sounds!
+
+@interface DetailsTableViewController ()
+{
+    AVAudioPlayer *_audioPlayerDoorbell;
+    AVAudioPlayer *_audioPlayerPhoneRinging;
+    AVAudioPlayer *_audioPlayerVintagePhoneRinging;
+    AVAudioPlayer *_audioPlayerTongueRolling;
+    AVAudioPlayer *_audioPlayerTwoToneDoorbell;
+    AVAudioPlayer *_audioPlayerTelephoneRing;
+    AVAudioPlayer *_audioPlayerButtonClickOn;
+
+}
+
+@end
+
+@implementation DetailsTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+#pragma mark - Ringtones
+
+    // this adds ringtone sound
+    NSString *path = [NSString stringWithFormat:@"%@/Doorbell.mp3", [[NSBundle mainBundle] resourcePath]];
+    NSURL *soundUrl = [NSURL fileURLWithPath:path];
+    // Create audio player object and initialize with URL to sound
+    _audioPlayerDoorbell = [[AVAudioPlayer alloc] initWithContentsOfURL:soundUrl error:nil];
```

Comment:
FYI you can use `NSBundle -URLForResource:withExtension:` ([docs](https://developer.apple.com/library/ios/documentation/Cocoa/Reference/Foundation/Classes/NSBundle_Class/index.html#//apple_ref/occ/instm/NSBundle/URLForResource:withExtension:)) to do this in fewer steps and without an explicit dependency on file paths:

```objective-c
    NSURL *soundUrl2 = [[NSBundle mainBundle] URLForResource:@"PhoneRinging" withExtension:@".mp3"];
    _audioPlayerPhoneRinging = [[AVAudioPlayer alloc] initWithContentsOfURL:soundUrl2 error:nil];
```

This way, you can clean up your audio files into a subdirectory (group) within your bundle, like `/Sounds`, so that they don't clutter up your project list:

![screen shot 2015-08-17 at 6 47 11 pm](https://cloud.githubusercontent.com/assets/709100/9318289/6acef79c-4510-11e5-816a-0cac639b4015.png)